newspack-sacha/sass/style.scss

```
@@ -86,6 +88,7 @@ div.wpnbha .article-section-title,
 }
 
 .accent-header,
+.widgettitle,
```

The borders should just be applied to the `.widgettitle` class when it appears in the sidebar:

```suggestion
#secondary .widgettitle,
```